confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java

```diff
@@ -204,6 +204,7 @@ public TSStatus setTTL(SetTTLReq req) {
             .getStorageGroupSchema()
             .setTTL(req.getTTL());
         result.setCode(TSStatusCode.SUCCESS_STATUS.getStatusCode());
+        result.setMessage("StorageGroup does not exist");
```

Comment:
TSStatusCode.SUCCESS_STATUS
and 
result.setMessage("StorageGroup does not exist");

why？

Reply:
My mistake

Reply:
fixed~